index.qmd

```diff
@@ -23,6 +23,8 @@ format:
     transition: fade
 date: 2025-07-25
 engine: julia
+julia:
+    exeflags: ["+nightly"]
```

Comment:
Ah, fun, my `julia` is a local build from source, not `juliaup`.  I can deal with this but we have to install juliaup in CI, too

Reply:
oh I have both, but for me the build from source is managed via `juliaup` (I call it `julia +julia`)